pkg/kubelet/server/stats/types.go

```diff
@@ -104,6 +95,10 @@ type PodReference struct {
 
 // NetworkStats contains data about network resources.
 type NetworkStats struct {
+	// The time at which these stats were updated.
+	Time unversioned.Time `json:"time"`
+	// The time at which data collection for the cumulative stats was (re)started.
+	StartTime unversioned.Time `json:"startTime"`
```

Comment:
See below re: network stats. As for CPU / memory start time, I could go either way... I feel like putting the start time on the Cpu / memory stats object gives a little more flexibility to have different types of metrics in the future & disambiguates (e.g. container StartTime is not necessarily valid for UserDefinedMetrics). It's also a little more consistent to keep it next to the Time, but honestly I could go either way.